templates/webtest/demo_viewers/static_example.html

```diff
@@ -0,0 +1,177 @@
+<!doctype html>
+<!-- See discussion at https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058&p=17114 -->
+<html>
+<head>
+
+    <link href="https://cowfish.openmicroscopy.org/webmerge/static/omeroweb.viewer.min.css" type="text/css" rel="stylesheet"></link>
```

Comment:
I'd recommend to make a comment that is up to developer to come up with versioning strategy as that url will be cached in client web browser and may be outdated when OMERO.server is upgraded. Maybe we could add small JS script to put time-stamp in the end of min filed to make sure this is not cached by web browser